test/Interpreter/llvm_link_time_opt.swift

```diff
@@ -1,7 +1,7 @@
 // UNSUPPORTED: OS=windows-msvc
 // static library is not well supported yet on Windows
 
-// REQUIRES: lld_lto
+// XFAIL: OS=linux-android, OS=linux-androideabi
```

Comment:
This started failing recently on the community Android CI, as the LLVM 14 toolchain in the Android LTS NDK 25c doesn't support opaque pointers much:
```
error: Opaque pointers are only supported in -opaque-pointers mode (Producer: 'LLVM13.0.0git' Reader: 'LLVM 14.0.6git')
```
That flag wasn't added till clang 15, llvm/llvm-project@d69e9f9d89783, so this won't work till the next NDK, which will include clang 17. It currently works natively in the Termux app, where I use LLVM 16.